src/chesster.js

```diff
@@ -1064,7 +1064,7 @@ function(bot, message) {
         if(
             !_.isEqual(result.white.id, message.user) &&
             !_.isEqual(result.black.id, message.user) &&
-            !message.player.isModerator()
+            !message.player.isModerator(message)
```

Comment:
I don't understand why message was required here. But I also haven't looked at where this is defined.